osu.Game/Scoring/Legacy/LegacyScoreEncoder.cs

```diff
@@ -29,7 +30,7 @@ public class LegacyScoreEncoder
         /// <summary>
         /// The first stable-compatible YYYYMMDD format version given to lazer usage of replays.
         /// </summary>
-        public const int FIRST_LAZER_VERSION = 30000000;
+        public const int FIRST_LAZER_VERSION = 30000001;
```

Comment:
I don't think you want to change the `const` but the one above it instead (`LATEST_VERISION`). Changing this one will break some other code that is relying on knowing the first-ever lazer replay version ie:

https://github.com/ppy/osu/blob/da93849b8084b7bc0fb057704ee1a7a1606bf82f/osu.Game/Rulesets/Objects/Legacy/ConvertHitObjectParser.cs#L348